package.json

```diff
@@ -22,7 +22,7 @@
     "postpublish": "pinst --disable",
     "lint": "eslint . --ext .js,.ts --ignore-path .gitignore",
     "fix": "npm run lint -- --fix",
-    "postinstall": "opencollective-postinstall || exit 0"
+    "_postinstall": "opencollective-postinstall || exit 0"
```

Comment:
@typicode q: is this a typo?, if not this hook will not be triggered

Reply:
It’s prefixed with a _ to prevent postinstall to run in development. But it’s renamed to postinstall by pinst during prepublish.

But I can understand it’s confusing :)